eldritch/encounters.py

```diff
@@ -367,6 +367,136 @@ def Isle7(char):
+def Hospital6(char):
+  check = events.Check(char, "will", -1)
+  item = events.Draw(char, "unique", 1)
+  fail1 = events.Lose(char, {"sanity":1})
+  fail2 = events.ForceMovement(char, "Uptown")
+  fail = events.Sequence([fail1, fail2], char)
+  return events.PassFail(char, check, item, fail)
+def Hospital7(char):
+  check = events.Check(char, "lore", 0)
+  clue = events.Gain(char, {"clues": 1})
+  return events.PassFail(char, check, clue, events.Nothing(char))
+
+def Woods1(char):
+  box = events.Check(char, "luck", 0)
+  foot = events.Lose(char, {"sanity": 1})
+  common = events.Draw(char, "common", 1)
+  unique = events.Draw(char, "unique", 1)
+  jewelry = events.Gain(char, {"dollars": 1})
+  cond = events.Conditional(char, box, "successes", {0: foot, 1: common, 2: unique, 3: jewelry})
+  open_box = events.Sequence([box, cond], char)
+  return events.BinaryChoice(char, "Open the locked box?", "Yes", "No", open_box, events.Nothing(char))
+def Woods2(char):
+  check = events.Check(char, "sneak", -1)
+  make_check = events.PassFail(char, check, events.Nothing, events.Loss(char, {"stamina":2}))
+  leave = events.ForceMovement(char, "Uptown")
+  return events.Sequence([make_check, leave], char)
+def Woods3(char):
+  check = events.Check(char, "sneak", -2)
+  shotgun = events.DrawSpecific(char, "common", "Shotgun")
+  fail = events.Sequence([events.Loss(char, {'stamina': 2}), events.ForceMovement(char, "Uptown")], char)
+  return events.PassFail(char, check, shotgun, fail)
+def Woods4(char):
+  check = events.Check(char, "luck", -1)
+  bushwhack1a = events.ItemChoice(char, "Choose first item to lose")
+  bushwhack1b = events.DiscardSpecific(char, bushwhack1a)
+  bushwhack1c = events.ItemChoice(char, "Choose second item to lose")
+  bushwhack1d = events.DiscardSpecific(char, bushwhack1d)
+  bushwhack2 = events.Loss(char, {"stamina": 2})
+  bushwhack = events.Sequence([
+    bushwhack1a,
+    bushwhack1b,
+    bushwhack1c,
+    bushwhack1d,
+    bushwhack2,
+  ], char)
+  return events.PassFail(char, check, events.Nothing(char), bushwhack)
+def Woods5(char):
+  #TODO: Check whether you have food to give to the doggy
+  #prereq = events.ItemPrerequisite(char, "Food")
+  prereq = events.Nothing(char)
+  check = events.Check(char, "speed", -2)
+  #TODO: Check whether the ally is available in the deck
+  #dog = events.GainAllyIfAvailable(char, "Doggy", otherwise={"dollars": 3})
+  dog = events.Nothing(char)
+  give_food = events.DiscardSpecific(char, "Food")
+  catch = events.PassFail(char, check, dog, events.Nothing(char))
+  seq = events.Sequence([give_food, dog], char)
+  choose_give_food = events.BinaryChoice(char, "Give food to the dog?", "Yes", "No", seq, catch)
+  return events.PassFail(char, prereq, choose_give_food, catch)
+def Woods6(char):
+  #TODO: A Gate and a Monster appear
+  return events.Nothing(char)
+def Woods7(char):
+  choice = events.MultipleChoice(char, "Which would you like to gain?", ["A skill", "2 spells", "4 clues"])
```

Comment:
I have arbitrarily chosen 100 characters as the line limit.
```suggestion
  choice = events.MultipleChoice(
      char, "Which would you like to gain?", ["A skill", "2 spells", "4 clues"])
```